src/fees/FeeCalculatorBase.sol

```diff
@@ -28,20 +28,8 @@ abstract contract FeeCalculatorBase {
     /// @param amount The amount with the fee included
     /// @return The calculated fee
     function calculateFee(uint256 amount) public view returns (uint256) {
-        return (amount * feeRate) / (_BPS_BASE + feeRate);
-    }
-
-    /// @notice Calculate the fees for the given array of amounts
-    /// @param amounts An array of amounts with the fee included
-    /// @return An array of calculated fees
-    function calculateFee(uint256[] memory amounts) public view returns (uint256[] memory) {
-        for (uint256 i; i < amounts.length; ) {
-            amounts[i] = (amounts[i] * feeRate) / (_BPS_BASE + feeRate);
-            unchecked {
-                ++i;
-            }
-        }
-        return amounts;
+        uint256 feeRate_ = feeRate;
+        return (amount * feeRate_) / (_BPS_BASE + feeRate_);
```

Comment:
```suggestion
        uint256 _feeRate = feeRate;
        return (amount * _feeRate) / (_BPS_BASE + _feeRate);
```
As internal coding style said `Local variables has prefix _ if conflicts with storage variable`

Reply:
Oh you are right. I confused with function parameters.